lib/suspenders/app_builder.rb

```diff
@@ -331,6 +331,7 @@ def configure_puma
 
     def set_up_forego
       copy_file 'sample.env', '.sample.env'
+      copy_file 'sample.env', '.env'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.